modin/pandas/test/test_io.py

```diff
@@ -1334,6 +1334,14 @@ def test_to_csv_with_index(self):
         ).set_index("key")
         eval_to_file(modin_df, pandas_df, "to_csv", "csv")
 
+    def test_read_csv_issue_5150(self):
+        pandas_df = pandas.DataFrame(np.random.randint(0, 100, size=(2**6, 2**6)))
+        pandas_df.to_csv("issue5150.csv", index=False)
```

Comment:
this creates a new file and doesn't clean it after the test is completed, right? It seems that the issue doesn't depend on that concrete dataset so we could use any other of the existing CSV files in `modin/pandas/test/data/`? 

If you'd still like to create a new file in the test, I would suggest doing it at least under `ensure_clean()` context or using one of our IO test functions that can create&clean test files.

Reply:
fixed